app/models/bgs_dependents/adult_child_attending_school.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module BGSDependents
+  class AdultChildAttendingSchool < Base
+    attribute :first, String
+    attribute :middle, String
+    attribute :last, String
+    attribute :suffix, String
+    attribute :ssn, String
+    attribute :birth_date, String
+    attribute :ever_married_ind, String
+
+    validates :first, presence: true
+    validates :last, presence: true
+
+    def initialize(dependents_application)
+      @dependents_application = dependents_application
+      @was_married = @dependents_application['student_address_marriage_tuition']['was_married']
+      @name_and_ssn = @dependents_application['student_name_and_ssn']
+
+      self.attributes = child_attributes
+    end
+
+    def format_info
+      attributes.with_indifferent_access
+    end
+
+    def address
+      @dependents_application['student_address_marriage_tuition']['address']
+    end
+
+    private
+
+    def child_attributes
+      # we will raise an error here if not #valid? when we merge in exception PR
+
+      {
+        ssn: @name_and_ssn['ssn'],
+        birth_date: @name_and_ssn['birth_date'],
+        ever_married_ind: @was_married == true ? 'Y' : 'N'
+      }.merge(@name_and_ssn['full_name'])
+    end
```

Comment:
Method name is ambiguous. Are we looking to access the `described_class`'s (human) child's attributes? Or are we accessing a hierarchy?

Reply:
I'm not sure if I follow your question but I will convey my intent, sorry if I'm stating the obv.

We are building a hash that we use in initialization to create attributes for the class.

How does `described_class_attribute_hash` sound?

Reply:
I've updated this method name to "described_class_attribute_hash".